packages/terra-framework-docs/src/terra-dev-site/doc/slide-panel/SlidePanel.1.doc.mdx

```diff
@@ -60,3 +60,13 @@ import SlidePanel from 'terra-slide-panel';
 
 ## Slide Panel Props Table 
 <SlidePanelPropsTable />
+
+## Testing
+
+The Slide Panel uses `uuid` which changes the component's description id dynamically. To mock it in jest test teams can use `jest.mock`.
```

Comment:
nit, even though it's copied can we update:
```suggestion
The Slide Panel uses `uuid` which changes the component's description id dynamically. To mock it in Jest tests teams can use `jest.mock`.
```

Reply:
Done: https://github.com/cerner/terra-framework/pull/1657/commits/a8531250eb7e1c1871507c8eb6241e14370af313